_dprhtml/js/dict_xml.js

```diff
@@ -343,7 +343,7 @@ async function sktRXML(no,add)
 
   if(add == 'right') return;
   if(add == true) {
-    //await sendDPPNXML(toVel(filein),true);
+    //await sendDPPNXML(DPR_translit_mod.toVel(filein),true);
```

Comment:
nice! letting the process do its stuff as opposed to thinking whether commented out or not, should be done or not etc.

basically the 'orthogonalization' thing we learn in dl.ai i.e. doing 1 thing at a time. planning, designing, implementing, testing are 3 separate things. we are not doing designing when implementing.

this does not stop the implementation from feeding back into the design - but it happens very explicitly/deliberately. as opposed to muddling up everything at the same time.